### PR TITLE
[Website] Increase contrast of input borders to meet accessibility requirements

### PR DESCRIPTION
**Pull request by RebeccaAnne**
# Related Issue

Fixes ADO [30943601](https://microsoft.visualstudio.com/OS/_queries/edit/30943601/?triage=true)

# Description

Updated color of input borders to meet contrast requirements. Contrast between input borders and white background is now 3.544:1

# How Verified

Verified contrast on website using dev tools and accessibility insights.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/6008)

### CODE REVIEW

File: source/nodejs/adaptivecards-visualizer/css/outlook.css [hunk 1]
```diff
@@ -201,7 +201,7 @@ a.ac-anchor:visited:active {
 }
 
 .ac-input.ac-textInput, .ac-input.ac-numberInput, .ac-input.ac-dateInput, .ac-input.ac-timeInput, .ac-input.ac-multichoiceInput {
-    border: 1px solid #DDDDDD;
+    border: 1px solid #888888;
```
Comment (almedina-ms) [inline source/nodejs/adaptivecards-visualizer/css/outlook.css:204]: is this the first shade of gray that complies with the contrast requirements? It looks like it's a big color change; @dclaux is there a reason why this color was chosen as the expected value?

https://www.color-hex.com/color/dddddd
https://www.color-hex.com/color/888888

Comment (dclaux) [reply source/nodejs/adaptivecards-visualizer/css/outlook.css:204]: The color used to match Fluent UI. It doesn't anymore, Fluent now uses `rgb(96, 94, 92)`:

image

**Comment by paulcam206** [comment]
@RebeccaAnne - do we still want to take this?

**Comment by RebeccaAnne** [comment]
Sev 3 isssue. Closing for now.

**Pull request closed without merging**